crit-go/gocrit/cmdhandler.go

```diff
@@ -0,0 +1,115 @@
+package gocrit
+
+import (
+	"crit-go/images"
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+func Check(e error) {
+	if e != nil {
+		fmt.Println(e)
```

Comment:
all errors have to be printed on stderr

Reply:
Resolved.

Reply:
I don't understand how you resolved it. The code is the same.

Reply:
just missed this file,other files were fixed.